docs/alerts/monitors/create-monitor.md

```diff
@@ -17,31 +17,31 @@ This topic shows you how to create a monitor.
 </details>
 
 
-## Open the New Monitor window
+## Step 1. Open the New Monitor window
```

Comment:
@kimsauce, I don't think you should make this "Step 1". Notice how the sections of the New Monitor dialog are numbered 1-5. To reduce confusion for readers, I think we need to align the numbers of the steps with the numbers of the dialog sections. What do you think?

Reply:
Oops, you're right! I was wondering why I hadn't labeled that "Step 1" in the first place. Thanks for pointing that out.